### DISCUSSION TITLE
Are GRPC exporters connecting insecurely by default?

### DISCUSSION THREAD

**Discussion by kahinton** [open-telemetry/opentelemetry-java-instrumentation#2266] [Q&A]
Hi All,

I've been working on utilizing the agent to send traces to a Jaeger backend sitting behind an ingress-nginx endpoint in a Kubernetes cluster. When I attempt to run my application, I keep getting the following error when it attempts to export spans:

```
Caused by: io.netty.handler.codec.http2.Http2Exception: First received frame was not SETTINGS. Hex dump for first 5 bytes: 485454502f
```

The nginx logs indicate that it is receiving a request like "PRI * HTTP/2.0", which causes it to respond with an HTTP/1.1 response. Using grpcurl I can reproduce the same issue when passing the -plaintext flag, however when letting it utilize it's default secure connection everything works fine. This is leading me to believe that the opentelemetry agent is making insecure requests by default. Is that true, and if so, how can it be configured to make secure connections? I've tried using the standard jaeger collector as well as their otlp collector along with the corresponding exporters in the agent with the same results.

**Comment by anuraaga** [comment]
Thanks @kahinton - we currently are not interpreting the environment variable correctly

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter

An endpoint with `http://` should be insecure, one with `https://` should be secure. I sent https://github.com/open-telemetry/opentelemetry-java/pull/2821 to fix
